Core/Python/get_device_list.py

```diff
@@ -55,61 +52,76 @@ def __init__(self, session_input, output_details):
             return
 
         self.__base_uri = 'https://%s' % self.__session_input["ip"]
-        self.__headers = {'content-type': 'application/json'}
 
         try:
-            if self.__authenticate_with_ome() is False:
+            AUTH_SUCCESS, HEADERS = self.__authenticate_with_ome()
```

Comment:
Better to make headers as a class variable so that it does not need to be passed to all the methods